Userland/Applets/Audio/main.cpp

```diff
@@ -76,25 +83,32 @@ class AudioWidget final : public GUI::Widget {
                 window()->resize(44, 16);
                 m_percent_box->set_tooltip("Hide percent");
             }
-            reposition_slider_window();
+            m_repositioning_required = 2;
```

Comment:
This looks like a hack! ;)
If I understand correctly, it forces the window to be re-positioned and re-painted/layouted twice on startup, since it's only ever set in the ctor.
I don't know why, but I'm sure there are reasons...
At least I expect a `bool` for the variable name, making it more descriptive seems needed.

Reply:
A boolean does not work. I need to reposition one frame after the rect change happened, because the rect change is reported incorrectly for up to one frame. And this setting happens in an on-change callback, so it gets called regularly.

Reply:
There surely is a more elegant way to achieve what you want. But without ripping the whole of WindowServer apart that's a compromise I guess.
Still, I'd recommend renaming the variable and maybe a comment explaining the ominous `2` right there.